### PR TITLE
Cleaning headers in structuralmechanics

### PR DESCRIPTION
**Pull request by RiccardoRossi**
here we attempt to simplify header dependencies in the StructuralMechanicsApplication. This shall hopefully allow to reduce compilation times.

This is NOT for Release 5.2

### CODE REVIEW

File: applications/StructuralMechanicsApplication/structural_mechanics_application.h [hunk 1]
```diff
@@ -79,8 +79,8 @@
 
 /* UTILITIES */
 // Cross sections
-#include "custom_utilities/shell_cross_section.hpp"
-#include "custom_utilities/multipoint_constraint_data.hpp"
+// #include "custom_utilities/shell_cross_section.hpp"
```
Comment (pooyan-dadvand) [inline applications/StructuralMechanicsApplication/structural_mechanics_application.h:84]: I would remove the lines

File: kratos/spaces/ublas_space.h [hunk 2]
```diff
@@ -810,7 +810,7 @@ class UblasSpace
     static void ParallelProductNoAdd(const MatrixType& A, const VectorType& in, VectorType& out)
     {
         //create partition
-        vector<unsigned int> partition;
+        boost::numeric::ublas::vector<unsigned int> partition;
```
Comment (pooyan-dadvand) [inline kratos/spaces/ublas_space.h:813]: 👍

**Review by pooyan-dadvand — APPROVED** [review]
It is ok for me
Only I would remove the commented lines directly

**Comment by roigcarlo** [comment]
Since release branch is already created, can we merge this after fixing conflicts?

**Pull request merged**